docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -215,54 +213,125 @@ for more information about the environment variables.
 
 The index name to write events to. The default is
 +"{beatname_lc}-%\{[beat.version]\}-%\{+yyyy.MM.dd\}"+ (for example,
-+"{beatname_lc}-{version}-2017.04.26"+). If you change this setting, you also
++"{beatname_lc}-{version}-{localdate}"+). If you change this setting, you also
 need to configure the `setup.template.name` and `setup.template.pattern` options
 (see <<configuration-template>>).
+ifndef::no_dashboards[]
+If you are using the pre-built Kibana
+dashboards, you also need to set the `setup.dashboards.index` option (see
+<<configuration-dashboards>>).
+endif::no_dashboards[]
+
+You can set the index dynamically by using a format string to access any event
+field. For example, this configuration uses a custom field, `fields.log_type`,
+to set the index: 
 
-When loading the pre-built Kibana dashboards via {beatname_uc},
-you also need to set the `setup.dashboards.index` option (see <<configuration-dashboards>>).
+["source","yaml",subs="attributes"]
+------------------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["http://localhost:9200"]
+  index: "%\{[fields.log_type]\}-%\{[beat.version]\}-%\{+yyyy.MM.dd}\" <1>
+------------------------------------------------------------------------------
 
+<1> We recommend including `beat.version` in the name to avoid mapping issues
+when you upgrade.
 
-===== `indices`
+With this configuration, all events with `log_type: normal` are sent to an 
+index named +normal-{version}-{localdate}+, and all events with
+`log_type: critical` are sent to an index named
++critical-{version}-{localdate}+.
 
-Array of index selector rules supporting conditionals, format string
-based field access and name mappings. The first rule matching will be used to
-set the `index` for the event to be published. If `indices` is missing or no
-rule matches, the `index` field will be used.
+TIP: To learn how to add custom fields to events, see the
+<<libbeat-configuration-fields,`fields`>> option.  
 
-Rule settings:
+ifndef::no-indices-rules[]
 
-*`index`*: The index format string to use. If the fields used are missing, the rule fails.
+See the <<indices-option-es,`indices`>> setting for other ways to set the index
+dynamically.
 
-*`mapping`*: Dictionary mapping index names to new names
+endif::no-indices-rules[]
 
-*`default`*: Default string value if `mapping` does not find a match.
 
-*`when`*: Condition which must succeed in order to execute the current rule.
+ifdef::deprecate_dashboard_loading[]
```

Comment:
Ohhhh good find! Took me a while to figure it out. Looks like @\simitt added it here: https://github.com/elastic/beats/pull/7675/files#diff-f1a89e6c3a1ce6f296f47ad83b989ff7R224. Back then, it came right after this statement:
```
If you are using the pre-built Kibana dashboards,
you also need to set the `setup.dashboards.index` option (see <<configuration-dashboards>>).
```

Since then, other stuff has been added to the file and pushed it down the page. 

I'll move this WARNING up the page, and add a statement that clarifies what the deprecation applies to.

Reply:
Fixed in 1a88b8c1:
<img width="766" alt="screen shot 2018-12-14 at 2 09 59 pm" src="https://user-images.githubusercontent.com/5618806/50032671-2bd98900-ffaa-11e8-856c-a5bd8df4ebb6.png">